packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/interact/Form.tsx

```diff
@@ -14,7 +15,7 @@ import { FunctionField } from "./FunctionField";
 export function Form() {
   const [hash] = useHashState();
   const { data, isFetched } = useWorldAbiQuery();
-  const [filterValue, setFilterValue] = useState("");
+  const [filterValue, setFilterValue] = useQueryState("functions-filter", { defaultValue: "" });
```

Comment:
what about just `functions`?

Reply:
True, a bit too verbose. Was also thinking that these should be named rather well or else if we have to update them, it would be a pain to manage outdated URLs.